### DISCUSSION TITLE
No trip responses for some agencies

### DISCUSSION THREAD

**Discussion by gsmarquez** [transitland/transitland#353] [Q&A] [answered]
Is trip data only available for certain agencies? I am looking at agencies in LA county and am only able to get a response from the Trips endpoint for some agencies. 

For example, I can get a response for Santa Monica Big Blue Bus (operator id: o-9q5c-bigbluebus) using https://transit.land/api/v2/rest/routes/r-9q59x-43/trips?apikey={}. 

But then with LA Metro (operator id: o-9q5-metro~losangeles) I don't get a response when I use https://transit.land/api/v2/rest/routes/r-9q5c-45/trips?apikey={}

**Comment by drewda** [comment, marked as answer]
Feed versions can be imported at various levels of completeness. To save space, Transitland can unload older feed versions.

In the case of the [Big Blue Bus feed](https://www.transit.land/feeds/f-9q5c-bigbluebus), the latest feed version is marked as fully imported (with a double check mark):

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/212369/160014467-e0c688c0-15f7-4667-9544-f60719d0b959.png">

In contrast, the [Metro bus feed](https://www.transit.land/feeds/f-9q5-metro~losangeles) is only marked as having agencies, stops, and routes imported (a single check mark; this means trips and stop_times have been unloaded):

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/212369/160014783-368adf4e-e074-4ae0-9394-7686a3eba572.png">


This is typically what only happens for older feed versions. I'm not sure exactly why this is the case for the Metro bus feed. We'll investigate. But in the meantime, this is the way to tell what contents are available through Transitland APIs for each feed and its feed versions.

**Reply by irees** [reply]
@gsmarquez As @drewda mentioned, we found the cause, and have now identified a fix. The fix will take a few days to process the remaining data; I'll try to remember to provide an update in this thread when it's complete 😅
